src/Uno.UI/UI/Xaml/FrameworkElement.Layout.wasm.cs

```diff
@@ -270,23 +270,7 @@ private void InnerArrangeCore(Rect finalRect)
 			OnLayoutUpdated();
 		}
 
-		internal Thickness GetThicknessAdjust()
-		{
-			switch (this)
-			{
-				case Controls.Border b:
-					return b.BorderThickness;
-
-				case Controls.Panel g:
-					return g.BorderThickness;
-
-				case Controls.ContentPresenter p:
-					return p.BorderThickness;
-
-				default:
-					return Thickness.Empty;
-			}
-		}
+		private protected virtual Thickness GetThicknessAdjust() => Thickness.Empty;
```

Comment:
I would change the name of this method if it becomes overridable. In the "layout.wasm.cs" file I can (barely) understand its purpose, but while in `Border` for instance I just didn't understand while it was not just named `GetBorderThickness`.